greynoise_test_rules/greynoise_noise_test.py

```diff
@@ -0,0 +1,22 @@
+from panther_greynoise_helpers import GreyNoiseAdvanced, GreyNoiseBasic
+
+
+# This rule was created to test GreyNoise helper functionality and should not be enabled
+def rule(event):
+    advanced_noise = GreyNoiseAdvanced(event)
```

Comment:
should we add a helper that can tell the following:
1. if grey noise is supported at all from the event
2. which version of grey noise is enabled?